Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```diff
@@ -1,11 +1,18 @@
-/* Style applied to the BocEnumValue. */
+.bocEnumValue
+{
+  --label-alignment-margin : 0.125em;
+  --icon-alignment-margin : 0.25em;
+  --negative-icon-alignment-margin : -0.25em;
+  --negative-disabled-label-alignment-margin : -4px;
```

Comment:
Please add a todo for this in RM-7968. I will need to tweak this for names and sizes (e.g. it may be some of those dimensions are actually dependent on other variables...)